src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -83,11 +83,12 @@ public class Session {
 
             var useridentity = User as WindowsIdentity;
             // In remote broker User Identity type is always WindowsIdentity
-            string suppressUI = (useridentity == null) ? string.Empty : "--suppress-ui ";
+            string suppressUI = (useridentity == null) ? string.Empty : "--rhost-suppress-ui ";
+            string isRepl = Id.StartsWithIgnoreCase("REPL") ? string.Empty : "--rhost-interactive ";
```

Comment:
It can be the exact same string check if need be, just on the other side of the wire (in `BrokerClient` presumably, or really anywhere the command line arguments for the host flow through before they get to the client). @AlexanderSher might have a more specific preference.